jmclient/jmclient/jsonrpc.py

```diff
@@ -97,11 +100,11 @@ def queryHTTP(self, obj):
 
                 data = response.read()
 
-                return json.loads(data)
+                return json.loads(data.decode('utf-8'), parse_float=Decimal)
```

Comment:
Note that the decimal change in `blockchaininterface.py` corresponds to this change in the json parser.

Reply:
Thanks